<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radio1">
        <property name="text">
         <string>Mouse in Volume Rendering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio2">
        <property name="text">
         <string>Mouse in Surface Rendering</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QVTKWidget" name="qVTK1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QVTKWidget" name="qVTK2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="coord">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
    <addaction name="actionInput_image_size"/>
    <addaction name="actionInput_the_number_of_image_series"/>
    <addaction name="actionInput_data_space"/>
   </widget>
   <widget class="QMenu" name="menuVolume_Rendering">
    <property name="title">
     <string>Volume Rendering</string>
    </property>
    <addaction name="actionRayCasting"/>
    <addaction name="actionTextureMapper2D"/>
    <addaction name="actionTextureMapper3D"/>
   </widget>
   <widget class="QMenu" name="menuSurface_Rendering">
    <property name="title">
     <string>Surface Rendering</string>
    </property>
    <addaction name="actionMarchingCubes"/>
    <addaction name="actionNewMarchingCubes"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuVolume_Rendering"/>
   <addaction name="menuSurface_Rendering"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionRayCasting">
   <property name="text">
    <string>RayCasting</string>
   </property>
  </action>
  <action name="actionTextureMapper2D">
   <property name="text">
    <string>TextureMapper2D</string>
   </property>
  </action>
  <action name="actionMarchingCubes">
   <property name="text">
    <string>MarchingCubes</string>
   </property>
  </action>
  <action name="actionTextureMapper3D">
   <property name="text">
    <string>TextureMapper3D</string>
   </property>
  </action>
  <action name="actionRead_image_series">
   <property name="text">
    <string>Read image series</string>
   </property>
  </action>
  <action name="actionInput_image_size">
   <property name="text">
    <string>Input image size</string>
   </property>
  </action>
  <action name="actionInput_the_number_of_image_series">
   <property name="text">
    <string>Input the number of image series</string>
   </property>
  </action>
  <action name="actionNewMarchingCubes">
   <property name="text">
    <string>NewMarchingCubes</string>
   </property>
  </action>
  <action name="actionInput_data_space">
   <property name="text">
    <string>Input data space</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>GUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
